kivy/__init__.py

```diff
@@ -331,14 +331,19 @@ def kivy_usage():
 
         mp_fork = None
         try:
-            mp_fork = opts['multiprocessing-fork']
-        except:
+            for opt, arg in opts:
+                if opt in '--multiprocessing-fork':
```

Comment:
This line doesn't look right - 'for arg in string' will check each character of the string. Should `in` just be `==`?